server/jobs/storage_backend.go

```diff
@@ -26,6 +102,6 @@ func (s *NoopStorageBackend) Read(key string) ([]string, error) {
 	return []string{}, nil
 }
 
-func (s *NoopStorageBackend) Write(key string, logs []string) (success bool, err error) {
-	return false, nil
+func (s *NoopStorageBackend) Write(key string, reader io.Reader) error {
+	return &StorageBackendNotConfigured{}
```

Comment:
why is this changing in implementation? Isn't this supposed to not error out and just do nothing?

Reply:
That's based on my previous [discussion](https://github.com/lyft/atlantis/pull/189#discussion_r810287188).

Reply:
I didn't think about the case where storage backend is not configured.

Reply:
Yup, reverted it back to how it was before.